lang/test/org/partiql/lang/syntax/SqlParserTestBase.kt

```diff
@@ -150,75 +144,42 @@ abstract class SqlParserTestBase : TestBase() {
-        val actualElement = unwrapQuery(actualStatment)
+    private fun checkEqualInIonSexp(actualStatement: PartiqlAst.Statement, expectedIonSexp: IonSexp, source: String) {
+        val actualElement = unwrapQuery(actualStatement)
         val actualIonSexp = actualElement.toIonElement().asAnyElement().toIonValue(ion)
 
         assertSexpEquals(expectedIonSexp, actualIonSexp, "AST, $source")
     }
 
-    private fun pigDomainAssert(actualExprNode: ExprNode, expectedSexpAst: SexpElement) {
-        val actualStatement = actualExprNode.toAstStatement()
-        val actualElement = unwrapQuery(actualStatement)
-
-        // Check equal for actual and expected in transformed astStatement: astStatment -> SexpElement -> astStatement
-        // Check round trip for actual: SexpElement -> astStatement -> SexpElement
-        assertRoundTripIonElementToPartiQlAst(actualElement, expectedSexpAst)
-
+    private fun pigDomainAssert(actualStatement: PartiqlAst.Statement, expectedElement: SexpElement) {
         // Check equal for actual and expected in SexpElement format
-        // Check round trip for actual: astStatement -> SexpElement -> astStatement
-        // Check equals for actual value after round trip transformation: ExprNode -> astStatement -> SexpElement -> astStatement -> ExprNode
-        assertRoundTripPartiQlAstToExprNode(actualStatement, expectedSexpAst, actualExprNode)
-    }
-
-    private fun assertRoundTripPartiQlAstToExprNode(actualStatement: PartiqlAst.Statement, expectedSexpAst: IonElement, actualExprNode: ExprNode) {
-        // Run additional checks on the resulting PartiqlAst instance
-
-        // None of our test cases are wrapped in (query <expr>), so extract <expr> from that out
         val actualElement = unwrapQuery(actualStatement)
-        assertEquals(expectedSexpAst, actualElement)
+        assertEquals(expectedElement, actualElement)
 
-        // Convert the the IonElement back to the PartiqlAst instance and assert equivalence
-        val transformedActualStatement = PartiqlAst.transform(actualStatement.toIonElement()) as PartiqlAst.Statement
-        assertEquals(actualStatement, transformedActualStatement)
+        // Check equal after transformation: PIG AST -> SexpElement -> PIG AST
+        assertRoundTripPigAstToSexpElement(actualStatement)
 
-        // Convert from the PIG instance back to ExprNode and assert the result is the same as actualExprNode.
-        val transformedActualExprNode = MetaStrippingRewriter.stripMetas(transformedActualStatement.toExprNode(ion))
-        assertEquals(MetaStrippingRewriter.stripMetas(actualExprNode), transformedActualExprNode)
-    }
-
-    private fun assertRoundTripIonElementToPartiQlAst(actualElement: SexpElement, expectedElement: SexpElement) {
-        // #1 We can transform the actual PartiqlAst element.
+        // Check equal for actual and expected in transformed astStatement: astStatment -> SexpElement -> astStatement
```

Comment:
nit: typo. `astStatment` -> `astStatement`

Reply:
Done in the next commit with message 'Fixed typo in comments'.